simple-example/src/main.py

```diff
@@ -14,15 +15,26 @@ def main():
 
     print("Starting example model")
 
-    # Read value from environment variable using the default value
-    # if it is not specified and convert the value to an integer
-    sequence_length = int(os.getenv("SEQUENCE_LENGTH", SEQUENCE_LENGTH_DEFAULT))
-    sequence_f0 = int(os.getenv("SEQUENCE_F0", SEQUENCE_F0_DEFAULT))
-    sequence_f1 = int(os.getenv("SEQUENCE_F1", SEQUENCE_F1_DEFAULT))
-
-    if sequence_length < 5:
-        raise Exception("Sequence_length is below the minimum value")
-
+    # Read values from environment variables but use default values if they don't exist
+    sequence_length = os.getenv("SEQUENCE_LENGTH", SEQUENCE_LENGTH_DEFAULT)
+    sequence_f0 = os.getenv("SEQUENCE_F0", SEQUENCE_F0_DEFAULT)
+    sequence_f1 = os.getenv("SEQUENCE_F1", SEQUENCE_F1_DEFAULT)
+    
+    # Check each of the values are in an acceptable format for this model
+    try:
+        sequence_length = int( sequence_length )
+    except ValueError:
+        sys.exit("Error: SEQUENCE_LENGTH must be a whole number")
+
+    if sequence_length <= 0:
+        sys.exit("Error: SEQUENCE_LENGTH must be greater than zero")
+    
+    try:
+        sequence_f0 = int(sequence_f0)
+        sequence_f1 = int(sequence_f1)
+    except ValueError:
+        sys.exit("Error: SEQUENCE_F0 and SEQUENCE_F1 must be whole numbers")
```

Comment:
I'd be inclined to mege this with the earlier try/except code you are using to check the sequence length. 

Or maybe pull this back out into a function like you had with the `is_int` function before. Would be nice to then pass the value and the variable name you can have a nice specific error message. Something like

```py
def to_int(value, name) -> int:
    try:
        return int(value)
    except ValueError:
        sys.exit(f"Error: {name} must be a whole number, not {value}")
```

Reply:
I had similar ideas, but just trying to balance it between too much and just enough code... I'll add it in for now.

Reply:
It is a tough balance to get right isn't it!